src/app/contexts/EventTrackingContext/index.jsx

```diff
@@ -27,6 +28,7 @@ export const EventTrackingContext = createContext({});
 const getCampaignID = pageType => {
   const campaignID = {
     [ARTICLE_PAGE]: 'article',
+    [MEDIA_ARTICLE_PAGE]: 'media-article',
```

Comment:
I believe this should be called `article-sfv` in analytics, just to confuse things 😅

Reply:
Soz, I forgot to change it back.